apps.gen.go

```diff
@@ -305,6 +322,30 @@ type GitSourceSpec struct {
 	Branch       string `json:"branch,omitempty"`
 }
 
+// InstanceSize struct for InstanceSize
+type InstanceSize struct {
```

Comment:
As mentioned in https://github.com/digitalocean/godo/pull/383, it might be nice to prefix the instance structs (e.g. `AppInstanceSize`) as it's a fairly generic term and could lead to confusion with other resources.

Reply:
Good call — updated in https://github.com/digitalocean/godo/pull/386/commits/2d1a2b6465e94b6d632538a60251b739927ce259